src/Xamarin.Android.Build.Tasks/Tests/Xamarin.Android.Build.Tests/BuildTest2.cs

```diff
@@ -1083,9 +1083,11 @@ public void BuildAfterMultiDexIsNotRequired ()
 		}
 
 		[Test]
-		public void CustomApplicationClassAndMultiDex ()
+		public void CustomApplicationClassAndMultiDex ([Values (true, false)] bool isRelease)
 		{
 			var proj = CreateMultiDexRequiredApplication ();
+			proj.IsRelease = isRelease;
+			proj.TrimModeRelease = TrimMode.Full;
```

Comment:
Does this test need to run with and without TrimMode.Full with both debug and release?